.travis/build.sh

```diff
@@ -38,19 +38,27 @@ mvn spotbugs:check
 # Run testsuite with java 8 only
 if [ ${JAVA_MAJOR_VERSION} -eq 1 ] ; then
 
-  docker pull oryd/hydra:v1.0.0
+  docker pull oryd/hydra:v1.8.5
+  docker pull quay.io/keycloak/keycloak:latest
+
   mvn test-compile spotbugs:check -e -V -B -f testsuite
 
   set +e
 
   clearDockerEnv
-  docker pull strimzi/kafka:latest-kafka-2.6.0
+  docker pull quay.io/strimzi/kafka:latest-kafka-2.7.0
+  mvn -e -V -B clean install -f testsuite -Pkafka-2_7_0
+  EXIT=$?
+  exitIfError
+
+  clearDockerEnv
+  docker pull quay.io/strimzi/kafka:latest-kafka-2.6.0
   mvn -e -V -B clean install -f testsuite -Pkafka-2_6_0
   EXIT=$?
   exitIfError
 
   clearDockerEnv
-  docker pull strimzi/kafka:latest-kafka-2.5.1
+  docker pull quay.io/strimzi/kafka:latest-kafka-2.5.1
```

Comment:
I guess you can maybe use `0.21.1` instead of latest?

Reply:
Right, should make fore a more deterministic test env. Will change this.